src/main/kotlin/no/nav/familie/ef/sak/vilkår/gjenbruk/GjenbrukVilkårService.kt

```diff
@@ -50,6 +50,40 @@ class GjenbrukVilkårService(
         behandlingSomSkalOppdateres: UUID,
         behandlingIdSomSkalGjenbrukeInngangsvilkår: UUID,
     ) {
+        val vilkårsVurderingerForGjenbruk =
+            utledVilkårsvurderingerForGjenbrukData(
+                behandlingSomSkalOppdateres,
+                behandlingIdSomSkalGjenbrukeInngangsvilkår,
+            )
+        secureLogger.info(
+            "${SikkerhetContext.hentSaksbehandlerEllerSystembruker()} gjenbruker vurderinger fra behandling $behandlingIdSomSkalGjenbrukeInngangsvilkår " +
+                "for å oppdatere vurderinger på inngangsvilkår for behandling $behandlingSomSkalOppdateres",
+        )
+        vilkårsvurderingRepository.updateAll(vilkårsVurderingerForGjenbruk)
+    }
+
+    @Transactional
+    fun gjenbrukInngangsvilkårVurdering(
+        behandlingId: UUID,
+        behandlingIdForGjenbruk: UUID,
+        vilkårId: UUID,
+    ): Vilkårsvurdering {
+        val vilkårsVurderingForGjenbruk =
+            utledVilkårsvurderingerForGjenbrukData(
+                behandlingId,
+                behandlingIdForGjenbruk,
+            ).first { it.id == vilkårId }
```

Comment:
Her kan vi komme opp i et problem at man prøver å gjenbruke for et barn som ikke fantes i forrige behandling. Da vil `.first` smelle

Reply:
Det samme dersom man gjenbruker sivilstandsvilkår, men sivilstanden er endret

Reply:
Vil gjerne ha noen tester på begge disse scenariene